main_test.go

```diff
@@ -2,18 +2,13 @@ package main
 
 import (
 	"encoding/json"
-	"fmt"
 	"testing"
 )
 
 func TestUnmarshallSearchResponse(t *testing.T) {
 	var searchData SearchResponse
```

Comment:
SearchResponse doesn't belong to main package. Maybe `model.SearchResponse`?

Reply:
I'm resolving test comments for now, since I didn't add tests to this project yet,